src/Business Foundation/App/NoSeries/src/GlobalNoSeries/NoSeriesTenant.Table.al

```diff
@@ -0,0 +1,64 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+table 1263 "No. Series Tenant"
+{
+    Caption = 'No. Series Tenant';
+    DataPerCompany = false;
+    ReplicateData = false;
+    DataClassification = SystemMetadata;
```

Comment:
```suggestion
    DataClassification = CustomerContent;
```

Reply:
> 

In my opinion it is not CustomerData but SystemMetadata